doc/user_guide/user_guide.md

```diff
@@ -392,6 +393,33 @@ column types when preparing the table.
 | list                 |                   | VARCHAR(n), CHAR(n)             |
 | map                  |                   | VARCHAR(n), CHAR(n)             |
 | struct               |                   | VARCHAR(n), CHAR(n)             |
+| union                |                   | VARCHAR(n), CHAR(n)             |
+
+#### Orc Union Type
+
+Unlike `struct` type, `union` type holds a tagged field values. Therefore, there
+are no fields names in the union type.
+
+```java
+unionSchema = TypeDescription.fromString("struct<column:uniontype<i:int,s:string>>");
+unionValue = new OrcUnion(unionSchema);
+unionValue.set(0, new IntWritable(13));
+unionValue.set(1, new Text("abc"));
+```
+
+In the above example union type, we can only read the values using the
+positional tag index. For example, to read the integer `unionValue.0`.
+
+When converting this type into an Exasol `VARCHAR` we decided to use the
+category name of inner types as field names.
+
+The above union value will be converted to a JSON string as following:
```

Comment:
Do you also save a map as a JSON? What about the list and struct? Maybe we should add  a couple of sentences about other complex types

Reply:
Ohh, this is one is special. In nested struct we have field names, `struct<name: string, age: int>` this will be `{"name": "abc", "age": 12}`. Similar for list into JSON list `[1,2,3]`. 

Unlike struct, union does not have field names, but "tags" which is indexes 0..n. In this case, we use the type name as a field.